test/groups/group_operation_action.jl

```diff
@@ -14,6 +14,9 @@ include("group_utils.jl")
     @test repr(A_left) == "GroupOperationAction($(repr(G)), LeftAction())"
     @test repr(A_right) == "GroupOperationAction($(repr(G)), RightAction())"
 
+    @test switch_direction(LeftAction()) == RightAction()
+    switch_direction(RightAction()) == LeftAction()
```

Comment:
```suggestion
    @test switch_direction(RightAction()) == LeftAction()
```

Reply:
Ah, yes. I am wondering by the way, why the group test coverage is so low, the tests here seem quite exhaustive, but then some do not seem to “hit” their lines.

Reply:
I don't know, I didn't check that in a long time. There are some real gaps in coverage there IIRC.

Reply:
Me neither, but at first glance, most things should be covered but aren't – so we might check somewhen, to get coverage a little further.